CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ include(ROCMInstallSymlinks)
 include(ROCMCreatePackage)
 include(CheckCXXCompilerFlag)
 
+set(MIOPEN_SQLITE_SCHEMA_VER "1.0.0" CACHE STIRNG "Version of the sqlite schema")
```

Comment:
Maybe this should be renamed to `MIOPEN_PERFDB_SCHEMA_VER` since this is the schema version for perfdb. 

And why does this need to be a cmake variable? I dont see any reason the user will override this when invoking cmake. Why not make the version a const variable in the source?

Reply:
I agree, the user does not need to over ride it. I would prefer to keep it in cmake since its a central location and makes management easier.

Reply:
I can put it in config.h, would that be ok ?

Reply:
Fixed